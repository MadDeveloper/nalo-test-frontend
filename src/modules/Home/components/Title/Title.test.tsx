import renderer from "react-test-renderer"
import { Title } from "."

describe("Home module > Title", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<Title>Lorem ipsum</Title>).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
