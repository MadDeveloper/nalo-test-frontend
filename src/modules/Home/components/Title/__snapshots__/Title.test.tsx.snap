// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home module > Title should match snapshot 1`] = `
<h1
  className="root"
>
  Lorem ipsum
</h1>
`;
